test/context.test.ts

```diff
@@ -167,9 +167,9 @@ describe("Context", () => {
       context = new Context(event, github, {} as any);
     });
 
-    it("gets a valid configuration", async () => {
+    it.only("gets a valid configuration", async () => {
```

Comment:
i assume this was a temporary intention?

Reply:
Yeah just ignore this PR for now, I need to rewrite the tests first so they don’t test implementation details